chart_updater/updater.py

```diff
@@ -41,33 +41,81 @@ def _build_commit_message(manifest: Manifest) -> str:
             parts.append(f"image {manifest.image_tag}")
         return f"Release of {chart_name} {', '.join(parts)}"
 
+    @staticmethod
+    def _build_branch_name(manifest: Manifest, cluster: str, release: str) -> str:
+        return f"chart-updater/{cluster}/{release}/{manifest.chart_name}-{manifest.chart_version}"
+
+    @staticmethod
+    def _build_pr_title(manifest: Manifest, cluster: str, release: str) -> str:
+        return f"Automated Upgrade: {cluster}/{release} - {manifest.chart_name} to {manifest.chart_version}"
+
+    @staticmethod
+    def _build_pr_body(
+        manifest: Manifest, cluster: str, namespace: str, release: str
+    ) -> str:
+        return f"""## Description
+Cluster:
+{cluster}
+
+Namespace:
+{namespace}
+
+Release:
+{release}
```

Comment:
I think this will look a bit nicer
```suggestion
**Cluster:** {cluster}
**Namespace:** {namespace}
**Release:** {release}
```